cmp/ag-grid/AgGridModel.js

```diff
@@ -45,32 +46,41 @@ export class AgGridModel {
 
     /**
      * @param {Object} [c] - AgGridModel configuration.
-     * @param {boolean} [c.compact] - true to render with a smaller font size and tighter padding.
+     * @param {string} [c.sizingMode] - one of large, standard, compact, tiny
```

Comment:
We could potentially make an enumeration here and do some validation.

Not sure about the idea of supporting arbitrary custom modes --- that seems confusing to me.